moveit_ros/visualization/planning_scene_rviz_plugin/CMakeLists.txt

```
@@ -1,17 +1,23 @@
 set(MOVEIT_LIB_NAME moveit_planning_scene_rviz_plugin)
 
-add_library(${MOVEIT_LIB_NAME}_core SHARED
+add_library(${MOVEIT_LIB_NAME} SHARED
   src/planning_scene_display.cpp
-  include/moveit/planning_scene_rviz_plugin/planning_scene_display.h)
-set_target_properties(${MOVEIT_LIB_NAME}_core PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME}_core moveit_rviz_plugin_render_tools ${catkin_LIBRARIES} ${OGRE_LIBRARIES} ${QT_LIBRARIES} ${Boost_LIBRARIES})
-
-add_library(${MOVEIT_LIB_NAME} SHARED src/plugin_init.cpp)
+  include/moveit/planning_scene_rviz_plugin/planning_scene_display.hpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME} ${MOVEIT_LIB_NAME}_core ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 
-install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

This line has to be reverted back as
```suggestion
install(DIRECTORY include/ DESTINATION include)
```